packages/cli/src/commands/test.js

```diff
@@ -41,18 +42,12 @@ export const builder = (yargs) => {
       type: 'boolean',
       default: true,
     })
-    .option('collectCoverage', {
+    .option('collect-coverage', {
```

Comment:
We can probably remove this now since it's the same argument for jest?

Reply:
I left it here, because it appears in the `--help` output, but can take out.